book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
```

Comment:
- [ ] I think it would be better if you named this array "rentals" instead of rental because it will store multiple rentals and I think it is better that way.